backend/trading/risk/portfolio.py

```
@@ -0,0 +1,316 @@
+        cumulative = np.cumprod(1 + returns)
+        running_max = np.maximum.accumulate(cumulative)
+        drawdowns = (running_max - cumulative) / running_max
+        return np.max(drawdowns)
+    
+    def record_daily_pnl(self):
+        """Record end of day P&L for statistics."""
+        if self.daily_pnl_history:
+            prev_nav = self.daily_pnl_history[-1]
+            daily_return = (self.nav - prev_nav) / prev_nav if prev_nav > 0 else 0
+        else:
+            daily_return = (self.nav / self.initial_capital) - 1
```

Ruff (0.14.8)

Local variable `daily_return` is assigned to but never used

Remove assignment to unused variable `daily_return`

(F841)